test/integration/data/utt-change-timezone.stdout

```diff
@@ -0,0 +1,21 @@
+
+---------------------- Wednesday, Aug 08, 2018 (week 32) -----------------------
+
+Working Time: 10h02 [11h12]
+Break   Time: 0h00 [0h00]
+
+----------------------------------- Projects -----------------------------------
+
+(10h02) : traveling, work
+
+---------------------------------- Activities ----------------------------------
+
+(7h42) : traveling
+(2h20) : work
+
+
+----------------------------------- Details ------------------------------------
+
+(2h20) 00:00-11:20 work
+(7h42) 11:20-11:02 traveling
```

Comment:
The `11:20` and `11:02` (and others) are not converted to the local timezone of the report. Not sure if we should. Open for discussion.